README.md

```diff
@@ -1,2 +1,35 @@
 # HW01
-Edit the readme
+
+### Biography of Nick
+
+Howdy! I'm Nick Elliott, a new DRSB recruit. I'll keep this short and sweet, 
+just like me. I grew up in the small town of Ramona, east of San Diego. 
+My passion for biology began with dinosaurs and Steve Erwin. I eventually 
+pursued this passion to San Diego Sate University, where I recieved my BS 
+degree. I had my first research experience in Dr. Tanya Renner's Lab. We 
+investgigated carnivorous plant evolution through their chitinase enzymes. 
+Interestingly, carnivory seems to have developed several times from a fungal 
+pathogen response. We looked at the Sundew: *Drosera capensis*
+
+![Image of Sundew](http://www.growsundews.com/sundews/capensis/Capensis_red_color_compare.JPG)
+
+After a year in the Renner Lab, I left to begin an internship with the 
+California Institute for Regenerative Medicine, CIRM. I completed this work at 
+the Salk Institute. There I worked under Dr. Diana Hargreaves and Dr. Fangjian 
+Gao to study the BAF complex. The BAF complex is a chromatin remodeler important
+for reguation in most tissue types. We focused on its role in neural 
+developement and mutations that contribute to intellectual diability. 
+Dr. Hargreaves was gracious enough to keep me on after the end of my internship 
+and I'm still working at the Salk now. Here's a [link to the lab!](https://hargreaves.salk.edu/) 
```

Comment:
Very cool you're still able to work there even with the pandemic stopping a lot of other research.